CHANGELOG.md

```diff
@@ -3,6 +3,29 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v1.1.4 - [2023-11-07]
+
+### `Added`
+
+### `Fixed`
+
+- [#306](https://github.com/nf-core/funcscan/pull/306) Added new parameter `annotation_prokka_retaincontigheaders` to allow prokka to retain the original contig headers/locus tag. (by @darcy220606)
+- [#307](https://github.com/nf-core/funcscan/pull/307) Fixed stability of deepARG tests by using Zenodo copy of database (❤️ to Gustavo Arango and Liqing Zhang for uploading, fix by @jfy133)
+- [#310](https://github.com/nf-core/funcscan/pull/310) Fixed error when supplying uncompressed input; added "fas" file extension for FASTA files. (by @tavareshugo)
+- [#311](https://github.com/nf-core/funcscan/pull/311) Merged pipeline template of nf-core/tools version 2.10. (by @jasmezz)
+
+### `Dependencies`
+
+| Tool          | Previous version | New version |
+| ------------- | ---------------- | ----------- |
+| AMRFinderPlus | 3.10.42          | 3.11.18     |
+| Bakta         | 1.7.0            | 1.8.2       |
+| MultiQC       | 1.1.4            | 1.1.5       |
```

Comment:
I am guessing typo and it should be 1.15? The newest available module is actually 1.17...

Reply:
Okay that is probably because this PR is around for a couple of weeks already :see_no_evil: I'd do a big module update in the next release and just update the typo here.